### DISCUSSION TITLE
Reference station for CT

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#385] [Ideas]
Some (public) channels have a very exact CT (from GPS or DCF77). Others (mostly private or local stations) might not care about it and even have default settings or wrong clock times at all (which differs from the actual time).

My idea is a menu option, which of the (currently) received RDS programs can be trusted to be very exact. Let's call this the CT reference.

For example if you turn on the new feature (in the first line) then a complete scan needs to be done in order to get the values for the PI/PS in the second line.

```
CT reference                                           on/off
CT reference station                              A203 _OE_3_
```

This CT should be synced to the internal clock then.

When you now scan through the band then all stations with a time difference are shown with `- x min` or `+ x min` (for example in different color) and the actual clock is displayed alternating. If the clock is the same there is no need to show that time offset.

**Comment by PE5PVB** [comment]
Well, you have crappy transmitter technicians over there ;-)

**Comment by PE5PVB** [comment]
Please tell me how you think one should configure this in a user-friendly way.

**Comment by PE5PVB** [comment]
Done in https://github.com/PE5PVB/TEF6686_ESP32/commit/798c9b43cfc2099f5d1e61773c9358979025c766

**Comment by andimik** [comment]
Works as expected. Can be closed.
